ioc/start.sh

```
@@ -70,43 +70,64 @@ generate_pvi_from_template() {
         /epics/pvi-defs/
 }
 
+generate_pvi_from_template() {
+    local template="$1"
+    local output_asset_namename="$2"
```

Shellcheck (0.11.0)

[warning] output_asset_namename appears unused. Verify use (or export if used externally).

(SC2034)